packages/_tests/test_packages_common.py

```diff
@@ -44,7 +46,14 @@ def test_parse_package(name: str) -> None:
 @pytest.mark.skip_refcount_check
 @pytest.mark.driver_timeout(60)
 @pytest.mark.parametrize("name", registered_packages())
-def test_import(name: str, selenium_standalone: _BrowserBaseRunner) -> None:
+@pytest.mark.benchmark(
+    max_time=3,
+    min_rounds=1,
+    timer=time.time,
```

Comment:
Better to use `time.perf_counter()` as  `time.time` is really imprecise (~16ms precision) on Windows though I guess we are not running there too much yet.

Reply:
Thanks for suggestion, opened a PR.